.github/CODEOWNERS

```diff
@@ -1,15 +1,25 @@
-# default code-owner
+# Default code-owner
 * @ethereum-optimism/security-reviewers
 
+# Code and script ownership
+
+## Go
 *.go @ethereum-optimism/go-reviewers
 /go.work @ethereum-optimism/go-reviewers
 /.golangci.yml @ethereum-optimism/go-reviewers
 /validation @ethereum-optimism/go-reviewers
 /superchain @ethereum-optimism/go-reviewers
-/superchain/extra/addresses @ethereum-optimism/security-reviewers
```

Comment:
Was `/superchain/extra/addresses` intentionally removed? Perhaps we split the fallback owners for the `/superchain` dir to `@ethereum-optimism/go-reviewers` for `*.go` + `go.mod` + `go.sum` files, and `@sbvegan @ZakAyesh @OPMattie` for the rest

Reply:
> That's more or less what I am doing in this PR except that for implementations that's more of a superchain-ops and chain upgrade thing than a chain onboarding thing? i.e. chains being added do not need to modify /superchain/implementations

So the contributing file says:

> Per superchain a set of canonical implementation deployments, per semver version, is tracked.
As default, an empty collection of deployments can be set:

I think this should be tweaked to clarify that it's the "governance approved standard implementations starting from `op-contracts/v1.3.0` and later" (i.e. MCP and later). In which case I agree with the `/superchain/implementations @ethereum-optimism/security-reviewers` assignment

Reply:
Per our slack conversation, we can keep the codeowners here as is, and make the above copy tweaks to clarify intent either in this PR or a future one

Reply:
Tracking in https://github.com/ethereum-optimism/superchain-registry/issues/164